docs/FAQ.md

```diff
@@ -19,6 +19,30 @@ image = Image.fromarray(numpy_image)
 image.save(name)
 ```
 
+## How do I make training faster?
+
+While it is impossible to anticipate the setup for all users, there are a few guidelines. First, a gpu-enabled processor is key. Training on CPU can be done, but it will take much longer (100x) and probably is only done if needed. Using google colab can be helpful, but somewhat annoying to use. Once on gpu, the config has a 'workers' argument. This connects to pytorch's dataloader. As the number of workers increase, data is fed to the GPU in parallel. Increase the worker argument slowly, we have found that the optimal number of workers vary by system. 
```

Comment:
While it is impossible to anticipate the setup for all users, there are a few guidelines. First, a GPU-enabled processor is key. Training on a CPU can be done, but it will take much longer (100x) and is probably only done if needed. Using Google Collaborate can be beneficial but also annoying. Once on the GPU, the configuration includes a "workers" argument. This connects to PyTorch's dataloader. As the number of workers increases, data is fed to the GPU in parallel. Increase the worker argument slowly, we have found that the optimal number of workers varies by system.